src/styleguide/introduction.md

```diff
@@ -0,0 +1,26 @@
+<style>
+.quote {
+  font-size: 20px;
+  line-height: 32px;
+  background-color: #ECFCFB;
+  margin-bottom: 22px;
+  padding: 30px;
+  border-radius: 8px;
+}
+</style>
+
+<hr style="margin: 30px 0 !important" />
+
+<div className="quote">"A Design System contains the knowledge that designers and developers need to create products faster and with consistency"</div>
+
+The The Store Style Guide consists in a collection of foundational elements (colors, text styles, icons), UI components and UX guidelines for their usage.
```

Comment:
Maybe wrap 'The Store Style Guid' with`<strong>` so it will stand out a bit more.